//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcdeg_TV_AVP.h
//
// Code generation for function 'abcdeg_TV_AVP'
//

#ifndef ABCDEG_TV_AVP_H
#define ABCDEG_TV_AVP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdeg_TV_AVP(double P_init, double V_init, double A_init, double P_wayp,
                   double V_wayp, double A_wayp, double A_max, double J_max,
                   double J_min, double T, creal_T t[28]);

#endif
// End of code generation (abcdeg_TV_AVP.h)
